DigitalLearningSolutions.Web/Views/MyAccount/EditDetails.cshtml

```diff
@@ -1,95 +1,117 @@
+    <h1 class="nhsuk-heading-xl" id="app-page-heading">Edit details</h1>
+  </div>
+</div>
+
+<form class="nhsuk-u-margin-bottom-8" method="post" novalidate asp-action="EditDetails" enctype="multipart/form-data">
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      @if (errorHasOccurred) {
+        <vc:error-summary order-of-property-names="@(new[] { nameof(Model.FirstName), nameof(Model.LastName), nameof(Model.Email), nameof(Model.Password) })"/>
+      }
+    </div>
+  </div>
+
+  <div class="hidden-submit">
+    <button name="action" class="nhsuk-button" value="save">Save</button>
+  </div>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      <vc:text-input asp-for="FirstName" label="First name" populate-with-current-value="true" type="text" spell-check="false" autocomplete="given-name" hint-text="" css-class="nhsuk-u-width-one-half"/>
+    </div>
+  </div>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      <vc:text-input asp-for="LastName" label="Last name" populate-with-current-value="true" type="text" spell-check="false" autocomplete="family-name" hint-text="" css-class="nhsuk-u-width-one-half"/>
+    </div>
+  </div>
+
+  <div class="nhsuk-grid-row divider">
+    <div class="nhsuk-grid-column-full">
+      <vc:text-input asp-for="Email" label="Email address" populate-with-current-value="true" type="text" spell-check="false" autocomplete="email" hint-text="" css-class="nhsuk-u-width-one-half"/>
+    </div>
+  </div>
+
+  @if (Model.IsDelegateUser) {
+    <input type="hidden" asp-for="IsDelegateUser"/>
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <vc:select-list asp-for="JobGroupId" label="Job group" value="@Model.JobGroupId.ToString()" hint-text="" css-class="nhsuk-u-width-one-half" default-option="Select a job group" select-list-options="@ViewBag.JobGroupOptions"></vc:select-list>
+      </div>
+    </div>
+
+    foreach (EditCustomFieldViewModel customField in ViewBag.CustomFields) {
+      <div class="nhsuk-grid-row @(ViewBag.CustomFields.IndexOf(customField) == ViewBag.CustomFields.Count - 1 ? "divider" : "")">
+        <div class="nhsuk-grid-column-full">
+          @if (customField.Options.Any())
+          {
+            <vc:select-list asp-for="@("Answer" + @customField.CustomFieldId)" label="@(customField.CustomPrompt + (customField.Mandatory ? "" : " (optional)"))" value="@customField.Answer" hint-text="" css-class="nhsuk-u-width-one-half" default-option="Select a @customField.CustomPrompt.ToLower()" select-list-options="@customField.Options"></vc:select-list>
+          } else {
+            <vc:text-input asp-for="@("Answer" + @customField.CustomFieldId)" label="@(customField.CustomPrompt + (customField.Mandatory ? "" : " (optional)"))" populate-with-current-value="true" type="text" spell-check="false" autocomplete="" hint-text="" css-class="nhsuk-u-width-one-half" />

```

Comment:
There are a lot of attribute values with some logic here so it might be worth separating this onto several lines like the text inputs on reset password page.
We should also be able to close the select-list VC with "/>" over </vc:select-list>